g2p/app.py

```diff
@@ -285,6 +285,7 @@ async def change_table(sid, message) -> None:
         mappings: List[Mapping] = []
         for lang1, lang2 in zip(path[:-1], path[1:]):
             transducer = make_g2p(lang1, lang2, tokenize=False)
+            assert isinstance(transducer, Transducer)
```

Comment:
Why this assertion? I like putting error messages with assertion statements so that they're self-documenting, and also informative if the AssertionError gets raised. i.e. `assert isinstance(transducer, Transducer), "AssertionError raised because____"`

Reply:
Agreed, this naked assert is not really better than the stack dump it would pre-empt.